plugins/enthusiast-common/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "enthusiast-common"
-version = "1.4.0"
+version = "1.4.1"
```

Comment:
That version needs to be aligned with Enthusiast's version - this is the core package that all custom packages link to, so if we introduce alternative numbering here, it's just confusing. We already have a gap with 1.4.1 where we didn't publish anything - we need to backfill it. 

Since we're not launching this PR as a hotfix, but as a part of the next version, let's not increase this version yet - we'll do that during the release.